setup.cfg

```diff
@@ -26,6 +26,11 @@ install_requires =
     networkx
 
 include_package_data = True
+packages =
```

Comment:
`findnamespace:` can be a little dangerous because you might have something that looks like a package but isn't (they give an example where a `tests` directory gets identified as a package). That's why I avoid it.